megamek/src/megamek/client/ui/baseComponents/AbstractDialog.java

```diff
@@ -113,22 +127,6 @@ protected void initialize() {
      */
     protected void finalizeInitialization() {
         pack();
-
-        // Escape keypress
-        final KeyStroke escape = KeyStroke.getKeyStroke(KeyEvent.VK_ESCAPE, 0);
-        getRootPane().getInputMap(JComponent.WHEN_IN_FOCUSED_WINDOW).put(escape, CLOSE_ACTION);
-        getRootPane().getInputMap(JComponent.WHEN_FOCUSED).put(escape, CLOSE_ACTION);
-        getRootPane().getActionMap().put(CLOSE_ACTION, new AbstractAction() {
-            private static final long serialVersionUID = 95171770700983453L;
-
-            @Override
-            public void actionPerformed(ActionEvent evt) {
-                cancelActionPerformed(evt);
-            }
-        });
-
-        addWindowListener(this);
-        setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);

```

Comment:
I had these here so that they could be overridden if a dev decided they needed to for an individual dialog. Is there are reason why we'd want to override the pairing here without requesting a custom finalization setup?